dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -40,92 +41,104 @@ Data
 	SXD23767 = Load(Filename='SXD23767.raw', LoadMonitors='Exclude')
 	# Set some UB with angles we can play with
 	SetUB(SXD23767, 1,1,2,90,90,120)
-	md_non_ortho = ConvertToDiffractionMDWorkspace(InputWorkspace='SXD23767', OutputDimensions='HKL')
+	SXD_MD_nonortho = ConvertToDiffractionMDWorkspace(InputWorkspace='SXD23767', OutputDimensions='HKL')
 
 .. figure:: ../../../../docs/source/images/MBC_PickDemo.png
    :alt: MBC_PickDemo.png
    :align: center
    :width: 75%
 
-- Create a PeaksWorkspace:
+- Create PeaksWorkspaces for the SXD data:
 
   - Open instrument viewer by right-clicking on the workspace``SXD23767``.
   - On the *Pick* tab, select the |PickTabAddPeakButton.png| "Add a single crystal peak" button.
   - Click on an intense bragg peak on the detectors, and then click on one or many of the intense peaks in the produced mini-plot. Repeat for a few different bragg peaks across the detectors.
   - Notice that this has produced a ``SingleCrystalPeakTable``.
-  - Rename the above workspace to ``peaks`` and in instrument viewer, select more peaks for a second peaksworkspace.
+  - Create another peak table which we will use to integrate (note in order to index these peaks we find the actual UB matrix which happens to be orthogonal - this is not a problem for test purposes)
+
+.. code-block:: python
+
+	FindSXPeaks(InputWorkspace='SXD23767', PeakFindingStrategy='AllPeaks', AbsoluteBackground=1500, \
+	    ResolutionStrategy='AbsoluteResolution', XResolution=500, PhiResolution=5, TwoThetaResolution=5, \
+	    OutputWorkspace='peaks')
+	FindUBUsingLatticeParameters(PeaksWorkspace='peaks', a=5.65, b=5.65, c=5.65, alpha=90, beta=90, gamma=90, FixParameters=True)
+    IndexPeaks(PeaksWorkspace='peaks')
```

Comment:
There seem to be an indentation issue with this block of code too.

Reply:
Yeh it doesn't like the line breaks so I will remove them